src/util/Permissions.js

```diff
@@ -64,6 +64,16 @@ class Permissions {
     return (this.bitfield & permission) === permission;
   }
 
+  /**
+   * Checks whether the bitfield has a permission, or any of multiple permissions.
+   * @param {PermissionResolvable} permissions Permission(s) to check for
+   * @param {boolean} [checkAdmin=true] Whether to allow the administrator permission to override 
+   */
+  any(permissions, checkAdmin = true) {
+    return (checkAdmin && this.has(this.constructor.FLAGS.ADMINISTRATOR)) ||
+      (this.bitfield & this.constructor.resolve(permissions, checkAdmin)) !== 0;
```

Comment:
```suggestion
      (this.bitfield & this.constructor.resolve(permissions)) !== 0;
```
resolve doesn't take checkAdmin param